app/views/layouts/avalon.html.erb

```diff
@@ -19,13 +19,20 @@ Unless required by applicable law or agreed to in writing, software distributed
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <title><%= render_page_title %></title>
+    <link rel="stylesheet" media="screen" href="https://fonts.googleapis.com/css?family=Open+Sans:400,700%7CPlayfair+Display:400,700%7CAbel" />
 
    <%= csrf_meta_tags %>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <%= favicon_link_tag %>
```

Comment:
Should maybe try to consolidate the two `favicon.ico` we have (/app/assets/ and /public folder into one?). 

I believe the default location the `favicon_link_tag` uses it the `/app/assets/` one, so you could just put in the actual markup to use the /public one, like you did below on line 31-35 (<link rel="icon" type="image/png" href="/favicon-16x16.png" sizes="16x16" />)